test/setup.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+# ------------------------------------------------------------------
+# [korthout] Setup test repo to be used to test the backport action
```

Comment:
not sure why your name appears here

Reply:
not sure either, I guess it was part of a template, but I'll remove it